docs/getting_started/installation.md

```diff
@@ -5,43 +5,208 @@ installation of the plugin and its dependencies. `uv` provides advanced
 dependency resolution which is required to properly install dependencies like
 `vllm` without overwriting critical dependencies like `torch`.
 
-First, clone the `vllm-spyre` repo:
+## Install `uv` (Pre-requisite)
 
-```sh
-git clone https://github.com/vllm-project/vllm-spyre.git
-cd vllm-spyre
-```
+You can [install `uv`](https://docs.astral.sh/uv/guides/install-python/) using `pip`:
 
-Then, install `uv`:
-  
 ```sh
 pip install uv
 ```
 
-Now, create and activate a new [venv](https://docs.astral.sh/uv/pip/environments/):
-  
+## Create a Python Virtual Environment
+
+Now create and activate a new Python (3.12) [virtual environment](https://docs.astral.sh/uv/pip/environments/):
+
 ```sh
-uv venv --python 3.12 --seed .venv
+uv venv --python 3.12 --seed .venv --system-site-packages
 source .venv/bin/activate
 ```
 
-To install `vllm-spyre` locally with development dependencies, use the following command:
+Since the full `torch_sendnn` stack is only available pre-installed in a base
+environment, we need to add the `--system-site-packages` to the new virtual
+environment to fully support the Spyre hardware. Pulling in the system site
+packages is not required for GPU-only installations.
+
+## Install vLLM with the vLLM-Spyre Plugin
+
+You can either install a released version of the vllm-Spyre plugin directly from
+[PyPI](https://pypi.org/project/vllm-spyre/) or you can install from source by
+cloning the [vllm-Spyre](https://github.com/vllm-project/vllm-spyre) repo from GitHub.
+
+=== "Release (PyPI)"
+
+    Note, to avoid any dependency resolution errors, we will install PyTorch
+    separately and tell `uv` to ignore any of it's dependencies while installing
+    the `vllm-spyre` plugin.
+    
+    ```sh
+    echo "torch; sys_platform == 'never'
+    torchaudio; sys_platform == 'never'
+    torchvision; sys_platform == 'never'
+    triton; sys_platform == 'never'" > overrides.txt
+    
+    uv pip install vllm-spyre --overrides overrides.txt
+    ```
+
+=== "Source (GitHub)"
+
+    First, clone the `vllm-spyre` repo:
+    
+    ```sh
+    git clone https://github.com/vllm-project/vllm-spyre.git
+    cd vllm-spyre
+    ```
+    
+    To install `vllm-spyre` locally with development dependencies, use the following command:
+    
+    ```sh
+    uv sync --frozen --active --inexact
+    ```
+    
+    To include optional linting dependencies, include `--group lint`:
+    
+    ```sh
+    uv sync --frozen --active --inexact --group lint
+    ```
+    
+    !!! tip
+        The `dev` group (i.e. `--group dev`) is enabled by default.
+
+## Install PyTorch
+
+Finally, `torch` is needed to run examples and tests. If it is not already installed,
+install it using `pip`:
+
+```sh
+pip install torch==2.7.1
```

Comment:
I might specify here that the `+cpu` package should be installed, since we also don't need any of the cuda deps that are included by default on linux installs. This requires `--index-url https://download.pytorch.org/whl/cpu` on linux, though on mac and windows the cpu package is the default one

Reply:
https://vllm-spyre--327.org.readthedocs.build/en/327/getting_started/installation.html#install-pytorch

<img width="722" height="361" alt="image" src="https://github.com/user-attachments/assets/2c7852a7-06e7-4c17-b14e-2094f6f31a8e" />